tests/subsys/sd/sdio/boards/arduino_nicla_vision_stm32h747xx_m7.conf

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2025 EXALT Technologies.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=3
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_SDHC=y
+CONFIG_SDMMC_STM32=y
+CONFIG_HEAP_MEM_POOL_SIZE=24576
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_CACHE_MANAGEMENT=y
```

Comment:
Same comment as above. Please seek for possible clean up of superfluous flag activations